### PR TITLE
Added section "Protect Redis from overgrowing"

### PR DESCRIPTION
**Pull request by letzya**
This caused a client a major issue and loss of analytics

### CODE REVIEW

File: tyk-docs/content/planning-for-production/planning-for-production.md [hunk 1]
```diff
@@ -120,6 +120,9 @@ You can calculate the right value using a straightforward formula:
 
 if the latency between Tyk and your Upstream is around 50ms, then a single connection can handle 1s / 50s = 20 requests. So if you plan to handle 2000 requests per second using Tyk, the size of your connection pool should be at least 2000 / 20 = 100. For example, on low-latency environments (like 5ms), a connection pool of 100 connections will be enough for 20k RPS.
 
+### Protect Redis from overgrowing
+Please read carefully through this [doc](https://site-dev.tykbeta.com/docs/basic-config-and-security/security/authentication-authorization/physical-token-expiry/) to make an *aware decision* about the TTL of your keys in Redis. If you don't set the lifetime a zero default means that keys will stay in Redis till you manually delete them, which is no issue if you have a process outside Tyk Gateway to handle it. If you don't and especially in scenarios that your flow creates many keys or access tokens for every user or even per call you Redis can quickly get cluttered with obsolete tokens and eventually affect the performences of the gateway.
```
Comment (sedkis) [inline tyk-docs/content/planning-for-production/planning-for-production.md]: switch
`https://site-dev.tykbeta.com/docs/basic-config-and-security/security/authentication-authorization/physical-token-expiry/)`
to
`/docs/basic-config-and-security/security/authentication-authorization/physical-token-expiry/)`

**Review by marksou — APPROVED** [review]
Looks good after amendment.

**Comment by letzya** [comment]
@marksou which one? I did a change at the same time you did :) I think you need to do it again - or text me to change

**Comment by marksou** [comment]
@letzya I changed the URL to a non hard coded one and capitalised Gateway. You amended the sentence about zero default. All looks fine now.

**Comment by letzya** [comment]
Thanks!!! appreciate it @marksou :) Let's wait for another engineer to approve

**Review by sedkis — APPROVED** [review]

File: tyk-docs/content/planning-for-production/planning-for-production.md (hunk 1 above)
Comment (jlucktay) [reply tyk-docs/content/planning-for-production/planning-for-production.md]: You reviewed an old commit mate; the latest one is fixed and doesn't have `https://site-dev.tykbeta.com` in front.

**Review by jlucktay — APPROVED** [review]

**Pull request merged**